templates/environment/partials/environment-manager-role.yml

```diff
@@ -37,10 +37,6 @@ EnvironmentManagerRole:
             "cloudwatch:DescribeAlarms"
           ]
           Resource: "*"
-          Condition: 
```

Comment:
This is because the alarms created by auto scaling are not propagated with copilot tags.

Reply:
so sad :(